Yoast/YoastSEO.js
# Adds passive voice assessment for Spanish

## Summary

This PR can be summarized in the following changelog entry:

Adds a passive voice assessment for Spanish.

## Relevant technical choices:

* The word list for the Spanish participles contains all words we want to match, with four different forms per word. We chose this way of matching participles because it yielded the best performance results when compared to matching with regexes. For future versions, we will investigate changing this list into a leaner data structure.

## Test instructions

This PR can be tested by following these steps:

- Check out the branch.
- Use the browserified example. Don't forget to run a `grunt build:js`.
-  Set the locale to `es_ES`.
- Make sure the following sentences are marked as passive:

  - `Una manzana es comida por mí.` (passive sentence)


- Make sure the following sentences are **not** marked as passive:
  - `Yo como una manzana.` (active sentence without participle)
  - `Es el capítulo preferido de varios miembros del equipo de producción.` (active sentence with auxiliary that's followed by an article)
  - `Fue un sentido monumental y grandilocuente.` (active sentence with a participle that's marked as non-passive by a direct precedence exception, such as the preceding article `un`)


Fixes #1370
